src/components/atoms/Toggler/Toggler.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import { Checkbox } from "../Checkbox";
+
+export interface TogglerProps {
+  toggled: boolean;
+  onChange: () => void;
+  containerClassName?: string;
+}
+
+export const Toggler: React.FC<TogglerProps> = ({
+  toggled,
+  onChange,
+  containerClassName = "",
```

Comment:
This will fallback to `undefined` and the html element will have a `className` of `undefined` where if it's an empty string the dom element will have no `className`

Reply:
A prop set to `undefined` is equivalent to a prop that isn't set at all. Setting `className` to `undefined` won't render an `"undefined"` class, it just won't set a class.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/c222264f9875e83b03a9a64b49c9ae1d57e9401c